index.html

```diff
@@ -64,6 +64,15 @@ <h2>Our Team</h2>
     <br/>
     Co-Founder, Full-Stack Engineer and Blockchain Specialist
   </p>
+
+  <p>
+    <span class="name">Timo Erdelt</span>
+    <a class="social-media-ref" href="https://www.linkedin.com/in/timoerdelt/" target="_blank"><img src="images/In-Black-66px-R.png" alt="linkedin reference"></a>
+    <a class="social-media-ref" href="https://github.com/tmrdlt" target="_blank"><img src="images/GitHub-Mark-64px.png" alt="github reference"></a>
+
+    <br/>
+    Full-Stack Engineer, Green IT, Functional programming, User-centered design
```

Comment:
To be consistent with upper and lower case on the rest of the website I would suggest the following:

```suggestion
    Full-Stack Engineer, Green IT, Functional Programming, User-Centered Design
```